addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -3240,6 +3250,10 @@ export class OdooEditor extends EventTarget {
         }
     }
 
+    _onTopWindowClick(){
+        this._updateToolbar(false);
+    }
```

Comment:
@Zynton @Mtaylorr, Couldn't we possibly handle this within `resetContent`? When the user clicks the `discard` button, the action triggers `resetContent` to remove the content.

Reply:
I'm not sure I understand here , this should be done whenever we click is outside the iframe not only on clicking discard button , let me know if I'm missing anything !

Reply:
Yes, you are right about it, but 16.0 clicking outside the side mail body removes the toolbar.

So, this task is for only 15.0 to 15.2?

Reply:
You are right , but actualy in 17.0 in studio we want the fix again since the same thing happens , so maybe we should keep it in 16.0 as well since we need to reintroduce it again in 17.0 ? @Zynton  what do you think

Reply:
Do you mean the bug doesn't occur in 16.0 but then returns in 17.0 but then only in studio?

Reply:
> Do you mean the bug doesn't occur in 16.0 but then returns in 17.0 but then only in studio?

Indeed

Reply:
Ah weird. Then indeed let's just integrate this fix regardless.